src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -91,149 +98,179 @@ const cspRulesDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+      message:
+        '"logging.events.request" and "logging.events.response" have been deprecated and will be removed ' +
         'in 8.0. To access request and/or response data moving forward, please enable debug logs for the ' +
         '"http.server.response" context in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
-    );
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+    });
   }
   return settings;
 };
 
-const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, deprecationHook) => {
   if (has(settings, 'logging.timezone')) {
-    log(
-      '"logging.timezone" has been deprecated and will be removed ' +
+    deprecationHook({
+      documentationUrl:
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+      message:
+        '"logging.timezone" has been deprecated and will be removed ' +
         'in 8.0. To set the timezone moving forward, please add a timezone date modifier to the log pattern ' +
         'in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
-    );
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+    });
   }
   return settings;
 };
 
-const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, deprecationHook) => {
   if (has(settings, 'logging.dest')) {
-    log(
-      '"logging.dest" has been deprecated and will be removed ' +
+    deprecationHook({
+      documentationUrl:
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+      message:
+        '"logging.dest" has been deprecated and will be removed ' +
         'in 8.0. To set the destination moving forward, you can use the "console" appender ' +
         'in your logging configuration or define a custom one. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
-    );
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx',
+    });
   }
   return settings;
 };
 
-const quietLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+const quietLoggingDeprecation: ConfigDeprecation = (settings, fromPath, deprecationHook) => {
   if (has(settings, 'logging.quiet')) {
-    log(
-      '"logging.quiet" has been deprecated and will be removed ' +
-        'in 8.0. Moving forward, you can use "logging.root.level:error" in your logging configuration. '
-    );
+    deprecationHook({
```

Comment:
@Bamieh would you mind adding a `documentationUrl` to the logging README here please? I didn't add it when handling logging deprecations but it would be great to get that in now.

Reply:
Done :)